controllers/managedocs_controller.go

```diff
@@ -115,24 +115,11 @@ func (r *ManagedOCSReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	managedOCSPredicates := builder.WithPredicates(
 		predicate.GenerationChangedPredicate{},
 	)
-	addonParamsSecretPredicates := builder.WithPredicates(
+	secretPredicates := builder.WithPredicates(
 		predicate.NewPredicateFuncs(
 			func(meta metav1.Object, _ runtime.Object) bool {
-				return meta.GetName() == r.AddonParamSecretName
-			},
-		),
-	)
-	pagerdutySecretPredicates := builder.WithPredicates(
-		predicate.NewPredicateFuncs(
-			func(meta metav1.Object, _ runtime.Object) bool {
-				return meta.GetName() == r.PagerdutySecretName
-			},
-		),
-	)
-	deadMansSnitchSecretPredicates := builder.WithPredicates(
-		predicate.NewPredicateFuncs(
-			func(meta metav1.Object, _ runtime.Object) bool {
-				return meta.GetName() == r.DeadMansSnitchSecretName
+				name := meta.GetName()
+				return name == r.AddonParamSecretName || name == r.PagerdutySecretName || name == r.DeadMansSnitchSecretName
```

Comment:
Can we multiline this? I am using a widescreen and I am still having trouble having it without scrolling.